spec/features/admin/configuration/dynamic_content_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+
+# Defines the test link string
+TEST_LINK = "ofn test link"
+
+# Defines the test href string. This one should contain all non-encoding characters, for a URL
+TEST_HREF = ":/?#@!$&'()*+,;=0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+
+describe 'add dynamic content to homepage' do
+	include AuthenticationHelper
+
+	before(:each) do
+		login_as_admin_and_visit spree.edit_admin_general_settings_path
+	end 
+	
+		
+	context 'edit the Content section', js: true do	
```

Comment:
we dont have a rule but I prefer " instead of '
there are more specs with " then ' I think.